apps/graphql/src/apps/itinerary/dataloaders/__tests__/Itineraries.test.js

```diff
@@ -13,10 +13,10 @@ const searchParamsOneWay = {
   },
   itinerary: {
     origin: {
-      ids: ['TG9jYXRpb246cHJhZ3VlX2N6'],
```

Comment:
Why has this changed?

Reply:
@tbergq Before, the unmasking of the IDs was done on the `parseParameters`. 

For simplicity we moved it out of the `parseParameters` to the resolvers (so you can unmask the IDs only in the case they are opaque i.e when inputting the IDs, but `Tequila Locations API` when searching for `slugs` gives you back an unmasked ID so we don't always unmask them, hence the reason of doing the conditional unmasking on the resolver rather than on the parseParameters). 

So now the input of the of the `parseParameters` are always unmasked IDs.

Reply:
Ok, got it 🙂